### PR TITLE
Inline legacyColumnMapping method in Druid

### PR DESCRIPTION
**Pull request by ebyhr**

### CODE REVIEW

**Comment by ebyhr** [comment]
CI #8432

File: plugin/trino-druid/src/main/java/io/trino/plugin/druid/DruidJdbcClient.java [hunk 1]
```diff
@@ -135,16 +154,82 @@ public ResultSet getTables(Connection connection, Optional<String> schemaName, O
     @Override
     public Optional<ColumnMapping> toColumnMapping(ConnectorSession session, Connection connection, JdbcTypeHandle typeHandle)
     {
+        Optional<ColumnMapping> mapping = getForcedMappingToVarchar(typeHandle);
+        if (mapping.isPresent()) {
+            return mapping;
+        }
+
+        // TODO (https://github.com/trinodb/trino/issues/9215) Implement proper type mapping and add test
```
Comment (hashhar) [inline plugin/trino-druid/src/main/java/io/trino/plugin/druid/DruidJdbcClient.java:162]: Maybe a link to https://github.com/trinodb/trino/issues/9215 here would be helpful - it mentions a few possible ways to ingest data into Druid (similar to how we test Pinot today).

**Review by hashhar — APPROVED** [review]
LGTM % comment

File: plugin/trino-druid/src/main/java/io/trino/plugin/druid/DruidJdbcClient.java (hunk 1 above)
Comment (ebyhr) [reply plugin/trino-druid/src/main/java/io/trino/plugin/druid/DruidJdbcClient.java:162]: Existing TODO comment already has a link to #9215?

Comment (hashhar) [reply plugin/trino-druid/src/main/java/io/trino/plugin/druid/DruidJdbcClient.java:162]: I'm going blind apparently. 😛

**Pull request merged**